packages/flutter/test/material/navigation_rail_test.dart

```diff
@@ -2349,6 +2349,50 @@ void main() {
     expect(indicator.width, 56);
     expect(indicator.height, 32);
   });
+
+  testWidgets('NavigationRailDestination has center aligned indicator - [labelType]=none', (WidgetTester tester) async {
+      // This is a regression test for
+      // https://github.com/flutter/flutter/issues/97753
```

Comment:
Formatting nit: I think this comment should be back two spaces, to line up with the `await _pumpNavigationRail` below.

Reply:
My pleasure! That's weird, I might have accidentally tabbed before commit, thanks